listener.py

```
@@ -151,13 +171,15 @@ def main() -> None:
                 grok_reply = get_grok_reply(prompt)
             except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)